lib/index.js

```diff
@@ -46,6 +46,16 @@ var epilogue = {
     if (!options.model)
       throw new Error('please specify a valid model');
 
```

Comment:
nitpick, I would probably just rewrite this as:
```
  options.sequelize = ensureSequelizeInstance(options.sequelize || this.sequelize);
```

and refactor the check into a function at the top of file:
```
function ensureSequelizeInstance(maybeInstance) {
  var instance = (maybeInstance.Sequelize) ? maybeInstance.Sequelize : maybeInstance;
  if (!_.every(requiredSequelizeAttrs, _.partial(_.has, instance)))
    throw new Error('invalid sequelize instance');

  return instance;
}
```